packages/playground/src/main.ts

```diff
@@ -97,6 +98,11 @@ async function main() {
   window.Y = Workspace.Y;
   window.std = std;
   window.ContentParser = ContentParser;
+  Object.defineProperty(globalThis, 'root', {
```

Comment:
Perhaps it's better to update `env.d.ts` so that we use `window.root` here?

Reply:
I just updated it.